CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ _None_
 
 ### Bug Fixes
 
-_None_
+- Fix metadata `po` generation for iOS projects removing the final `\n` [#498]
```

Comment:
Super pedantic nitpic: all our other entries end each list item with a `.`
```suggestion
- Fix metadata `po` generation for iOS projects removing the final `\n`. [#498]
```

Reply:
Damn auto-merge 😅

Reply:
I'll fix it as part of releasing a new version.